mono/metadata/object-internals.h

```diff
@@ -1234,7 +1234,7 @@ typedef enum {
 	MonoTypeBuilderFinished = 2
 } MonoTypeBuilderState;
 
-typedef struct {
+typedef struct _MonoReflectionTypeBuilder {
```

Comment:
This is unnecessary.

Reply:
Not entirely but I'll revisit.

Reply:
This instance is "needed" unless we revert to using MonoObject instead of MonoReflectionTypeBuilder.

Reply:
Is it necessary if you use `MonoReflectionTypeBuilder *builder` at https://github.com/mono/mono/pull/6196/files#diff-98cf043c47e9ac369f935d88ea2c0e5dR616?